generators/client/templates/angular/.prettierrc.ejs

```diff
@@ -1,7 +1,18 @@
 # Prettier configuration
+
 printWidth: 140
 singleQuote: true
-jsxBracketSameLine: false
-parser: typescript
-arrowParens: avoid
 tabWidth: 4
+useTabs: false
+overrides:
+- files: "src/**/{*.ts,*.tsx}"
+  options:
+    jsxBracketSameLine: false
+    arrowParens: avoid
+    parser: typescript
```

Comment:
This line is not mandatory. Prettier detects the parser needed.
@gmarziou. To simplify prettierrc, we could delete all "parser" lines?  What do you think about that? I asked me before commit, but I didn't dare delete that… 
Maybe it's better to have a more accurate prettierc? We have a complete syntax… Maybe it's better ^^! I believe it's better to see what we do… As you wish ;-). 

Note: parser will be renamed "language" (see https://github.com/prettier/prettier/issues/2846).

Reply:
Lets leave it like that as its more explicit

Reply:
@deepu105. I'm pretty sure, this syntax doesn't work. We could deduce that beccause we could see than 7fcd4b03a96bf10be6cd3ceb843b9c83f1f8a7cc works, but 7a854162f1d3729675bfabee2aa41666e3c3a3f0 fail. My assumption is that somewhere (but I don't know where) a command `$ yarn prettier --write --parser css "src/**/*.ts"` is launched. In my computer, when  in folder ngx-default-sample, when I run command `$ yarn prettier --write --parser css "src/**/*.ts"`, I have the same kind of error than in line 1526 of https://travis-ci.org/jhipster/generator-jhipster/jobs/365956929 (build of 7a854162f1d3729675bfabee2aa41666e3c3a3f0).

We could read at line 1526 of https://travis-ci.org/jhipster/generator-jhipster/jobs/365956929 than  
```
SyntaxError: (postcss) CssSyntaxError Unknown word (1:1)
> 1 | export * from './foo.service';
    | ^
  2 | export * from './foo-update.component';
  3 | export * from './foo-delete-dialog.component';
```
I don't understand this error beacause in folder "ngx-default-sample" there isn't file "foo.service", there isn't entity "foo". Have you an ideay why on remote Travis it tests folder "foo"?

As 7fcd4b03a96bf10be6cd3ceb843b9c83f1f8a7cc works, I could --hard reset to this commit ?

Or maybe I could try simply with a .prettierrc without explicit parser? Thanks 7fcd4b03a96bf10be6cd3ceb843b9c83f1f8a7cc, and with the result of the command « `$ yarn prettier --write --parser css "src/**/*.ts` » I believe it's better to let prettier choose the parser corresponding to the file extension. 
```typescript
printWidth: 140
singleQuote: true
tabWidth: 4
useTabs: false
overrides:
- files: "src/**/{*.ts,*.tsx}"
  options:
    jsxBracketSameLine: false
    arrowParens: avoid
```

What do you think about that?

Reply:
@deepu105 I'm going to test the syntax above with a new commit…